gensim/similarities/docsim.py

```diff
@@ -444,8 +448,20 @@ def save(self, fname=None, *args, **kwargs):
         self.close_shard()
         if fname is None:
             fname = self.output_prefix
+
         super(Similarity, self).save(fname, *args, **kwargs)
 
+    @classmethod
+    def load(self, fname, mmap=None):
```

Comment:
Class methods don't have `self`. Instead it's a class, called `cls` by convention.